7_inference/text_generation_inference.md

```diff
@@ -0,0 +1,137 @@
+# Text Generation Inference (TGI)
+
+Text Generation Inference (TGI) is a toolkit developed by Hugging Face for deploying and serving Large Language Models (LLMs). It's designed to enable high-performance text generation for popular open-source LLMs. TGI is used in production by Hugging Chat - An open-source interface for open-access models.
```

Comment:
I think it would be fair to mention other providers like vLLM and Ollama too, as being the same but different?